### DISCUSSION TITLE
Slow in transcribing?

### DISCUSSION THREAD

**Discussion by Jonseed** [openai/whisper#243] [Q&A] [answered]
Hi, I tried running the very simple Python script in the readme on a very short 7 second MP3, and it takes about 15 seconds to return the transcript. Is that speed normal? Or is there something wrong with my setup?

**Comment by jongwook** [comment, marked as answer]
Yes it's about the expected throughput, especially if you're running on a CPU. The model loading part is probably taking the majority of the time, and you can transcribe multiple files once the model is loaded.

**Reply by Jonseed** [reply]
It's probably the loading of the model, you're right. How do you tell if it is running on the CPU or GPU?

**Reply by jianfch** [reply]
`model.device`
